wrapper.js

```diff
@@ -50,6 +50,23 @@ function setupMethods (soljson) {
     } else {
       result = compileJSON(input, optimise);
     }
+
+    var res = JSON.parse(result);
+    if (res.errors) {
+      // TODO: Remove once the following is clarified:
+      // https://github.com/ethereum/solc-js/issues/53
+      // only leave the console.error
+
+      if (res.errors[0].indexOf('Warning') > 0) {
```

Comment:
I think `res.errors[0]` is wrong, it should check every single error for being a warning.